### PR TITLE
document to create hstore during upgrade

### PR DESCRIPTION
**Pull request by evgeni**
* [ ] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/doc-Upgrading_Project/topics/proc_upgrading-a-connected-project-server.adoc [hunk 1]
```diff
@@ -33,12 +33,26 @@ include::../common/modules/snip_warning-maintain-config-noop.adoc[]
 # {foreman-installer} --foreman-proxy-dns-managed=false \
 --foreman-proxy-dhcp-managed=false
 ----
-ifdef::katello[]
-. Optional: If you use PostgreSQL as an external database, install the `postgresql-evr` package on the PostgreSQL server:
```
Comment (evgeni) [inline guides/doc-Upgrading_Project/topics/proc_upgrading-a-connected-project-server.adoc:37]: I am re-purposing the `postgresql-evr` section here, as there is *no way* users can have arrived here without having `postgresql-evr` installed on their SQL servers.

**Review by ehelms — APPROVED** [review]

File: guides/doc-Upgrading_Project/topics/proc_upgrading-a-connected-project-server.adoc [hunk 2]
```diff
@@ -33,12 +33,26 @@ include::../common/modules/snip_warning-maintain-config-noop.adoc[]
...
+ifdef::katello,satellite,orcharino[]
```
Comment (mjivraja) [inline guides/doc-Upgrading_Project/topics/proc_upgrading-a-connected-project-server.adoc:36]: @AkshayGadhaveRH Need your help here. If installing `postgresql-contrib` applies to Satellite, Katello, and Orcharhino, why the changes of this PR are not visible for Orcharhino? You can see the bot-generated list of HTML files.

I saw the `master.adoc` but there is no condition at all for this file.

File: guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc [hunk 3]
```diff
@@ -62,6 +62,28 @@ endif::[]
 # {foreman-installer} --foreman-proxy-dns-managed=false \
 --foreman-proxy-dhcp-managed=false
 ----
+ifdef::katello,satellite,orcharino[]
```
Comment (mjivraja) [inline guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc:65]: @AkshayGadhaveRH Also here. We explicitly mention the applicable builds here but this whole file is `Satellite` specific.

File: guides/doc-Upgrading_Project/topics/proc_upgrading-a-connected-project-server.adoc (hunk 2 above)
Comment (mjivraja) [inline guides/doc-Upgrading_Project/topics/proc_upgrading-a-connected-project-server.adoc:36]:
```suggestion
ifdef::katello,satellite,orcharhino[]
```

File: guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc (hunk 3 above)
Comment (mjivraja) [inline guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc:65]:
```suggestion
ifdef::katello,satellite,orcharhino[]
```

**Review by mjivraja — CHANGES_REQUESTED** [review]
Ahh, it's a spelling mistake @evgeni :smile:

CC: @AkshayGadhaveRH

**Comment by evgeni** [comment]
Thanks for spotting this! Updated.

File: guides/doc-Upgrading_Project/topics/proc_upgrading-a-connected-project-server.adoc (hunk 1 above)
Comment (evgeni) [reply guides/doc-Upgrading_Project/topics/proc_upgrading-a-connected-project-server.adoc:37]: and removing it for 3.8+ in https://github.com/theforeman/foreman-documentation/pull/2579

File: guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc (hunk 3 above)
Comment (mjivraja) [reply guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc]: @evgeni What can be done here? Disconnected Guide is Satellite-exclusive as of now. We can remove the above condition for this PR at least.

Comment (evgeni) [reply guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc]: I can drop it completely, sure.

Comment (evgeni) [reply guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc]: But, fwiw, we already have conditionals in this file:
https://github.com/theforeman/foreman-documentation/blob/6021389e1d5d4dadfe7da9279a2bb63f37ffc37f/guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc?plain=1#L14

Comment (mjivraja) [reply guides/doc-Upgrading_Project/topics/proc_upgrading-a-disconnected-project-server.adoc]: Yes, this will be fixed soon. Thank you :+1:

**Review by mjivraja — APPROVED** [review]
LGTM

**Comment by evgeni** [comment]
To github.com:theforeman/foreman-documentation.git
   ff922a334..faa3626f4  3.6 -> 3.6
